spec/tic_tac_toe_spec.rb

```diff
@@ -2,7 +2,12 @@
 require './lib/tic_tac_toe'
 
 describe TicTacToe do
-  it 'determines true is true' do
-    expect(true).to eq true 
+  before(:each) do
+    @test_game=TicTacToe.new
+  end
+  describe "board" do
+    it 'an empty board has no marks' do
+      expect(@test_game.draw_board).to eq ""
+    end
```

Comment:
this test is failing in CI.

Reply:
I hadn't added my changes to this file to the previous git commit before pushing! This should be fixed now.